webapp/channels/src/components/widgets/modals/components/react_select_item.tsx

```diff
@@ -9,12 +10,17 @@ import type {BaseSettingItemProps} from './base_setting_item';
 import BaseSettingItem from './base_setting_item';
 
 export type Option = {
-    value: number;
-    label: string;
+    value: string;
+    label: string | ReactNode;
```

Comment:
`ReactNode` includes `string`
```suggestion
    label: ReactNode;
```